docs/api/en/renderers/WebGLMultipleRenderTargets.html

```diff
@@ -27,12 +27,18 @@ <h2>Examples</h2>
 		<h2>Constructor</h2>
 
 
-		<h3>[name]([param:Number width], [param:Number height], [param:Number count])</h3>
+		<h3>[name]([param:Number width], [param:Number height], [param:Number count], [param:Object options])</h3>
 
 		<p>
 		[page:Number width] - The width of the render target. <br />
 		[page:Number height] - The height of the render target.<br />
 		[page:Number count] - The number of render targets.
+
```

Comment:
You need to add a `<br />` at the end of the `count` line. Right now there is no line break, see:

https://raw.githack.com/zalo/three.js/fix-webglmultiplerendertargets-add-options/docs/index.html#api/en/renderers/WebGLMultipleRenderTargets

Reply:
Sorry about that (and thank you for the githack trick; I didn't realize it worked with the documentation too!)